beacon-chain/sync/validate_light_client.go

```diff
@@ -64,12 +70,12 @@ func (s *Service) validateLightClientOptimisticUpdate(ctx context.Context, pid p
 		return pubsub.ValidationIgnore, nil
 	}
 
-	if !lightclient.IsBetterOptimisticUpdate(newUpdate, s.lcStore.LastOptimisticUpdate()) {
+	if !proto.Equal(newUpdate.Proto(), currentUpdate.Proto()) {
 		log.WithFields(logrus.Fields{
 			"attestedSlot":       fmt.Sprintf("%d", newUpdate.AttestedHeader().Beacon().Slot),
 			"signatureSlot":      fmt.Sprintf("%d", newUpdate.SignatureSlot()),
 			"attestedHeaderRoot": fmt.Sprintf("%x", attestedHeaderRoot),
-		}).Debug("Newly received light client optimistic update ignored. current update is better.")
+		}).Debug("Invalid light client optimistic update received. Ignoring.")
```

Comment:
```suggestion
		}).Debug("Received light client optimistic update is different from the local one. Ignoring.")
```
I wouldn't say it's invalid, because to me it means that some part of it is incorrect (e.g. a required field is empty)